### PR TITLE
Enable rpo

### PR DESCRIPTION
**Pull request by dilipped**

### CODE REVIEW

File: gslib/commands/mb.py [hunk 1]
```diff
@@ -145,7 +147,13 @@
                          "enforced", objects in this bucket cannot be made
                          publicly accessible. Default is "unspecified".
 
-""")
+  --rpo setting          Specifies the `replication setting <https://cloud.google.com/storage/docs/turbo-replication>`_.
+                         Valid values are {rpo_values}. If unspecified,
```
Comment (starsandskies) [inline gslib/commands/mb.py:151]: My understanding is it's a bit more complicated, which we should probably convey here.

Suggestion: Specifies the `replication setting <https://cloud.google.com/storage/docs/turbo-replication>`_.
This flag is not valid for single-region buckets, and multi-region buckets only accept a value of DEFAULT. Valid values for dual region buckets are {rpo_values}. If unspecified, DEFAULT is applied for dual-region and multi-region buckets.

File: gslib/commands/rpo.py [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+VALID_RPO_VALUES = ('ASYNC_TURBO', 'DEFAULT')
+VALID_RPO_VALUES_STRING = '({})'.format('|'.join(VALID_RPO_VALUES))
+
+_SET_SYNOPSIS = """
+  gsutil rpo set {} gs://<bucket_name>...
+""".format(VALID_RPO_VALUES_STRING)
+
+_GET_SYNOPSIS = """
+  gsutil rpo get bucket_url...
```
Comment (starsandskies) [inline gslib/commands/rpo.py:40]: This and the _SET_SYNOPSIS appear inconsistent with regards to the bucket name/URL?

File: gslib/commands/mb.py (hunk 1 above)
Comment (dilipped) [reply gslib/commands/mb.py]: Done.

File: gslib/commands/rpo.py (hunk 2 above)
Comment (dilipped) [reply gslib/commands/rpo.py]: Done.

File: gslib/commands/mb.py [hunk 3]
```diff
@@ -147,11 +147,12 @@
                          "enforced", objects in this bucket cannot be made
                          publicly accessible. Default is "unspecified".
 
-  --rpo setting          Specifies the `replication setting <https://cloud.google.com/storage/docs/turbo-replication>`_.
-                         Valid values are {rpo_values}. If unspecified,
-                         the DEFAULT setting is applied for dual-region and
-                         multi-region buckets and
-                         None for single-region buckets.
+  --rpo setting          Specifies the replication setting <https://cloud.google.com/storage/docs/turbo-replication>_.
```
Comment (starsandskies) [inline gslib/commands/mb.py:150]: I think you'll still need the ` tickmarks for the linking to work?

**Comment by starsandskies** [comment]
LGTM

File: gslib/commands/mb.py (hunk 3 above)
Comment (dilipped) [reply gslib/commands/mb.py]: Good catch!

**Review by thomasmaclean — APPROVED** [review]

**Pull request merged**